autopush/router/simple.py

```diff
@@ -61,9 +61,15 @@ def amend_endpoint_response(self, response, router_data):
         """Stubbed out for this router"""
 
     def stored_response(self, notification):
+        self.metrics.gauge("notification.message_data",
```

Comment:
these would be simplepush only as webpush's router overrides these methods.

Would this stat be interesting on external bridges too? If so it might make sense for the WebHandlers to be emitting these